### DISCUSSION TITLE
fields in pdf

### DISCUSSION THREAD

**Discussion by MixTecnico** [YetiForceCompany/YetiForceCRM#15776] [General] [answered]
I am trying to make a weekly report of the cases in my helpdesk

but at the time of making the pdf the values are repeated, example tickets from 23-45

in all the columns the information of ticket 23 appears

How could I differentiate each field, is this possible to make a personalized sequence in each column...

I hope you understand me

**Comment by mariuszkrzaczkowski** [comment]
show some example

**Reply by MixTecnico** [reply]
here I know how the pdf is assembled, I know that using $(recordsList : HelpDesk)$ shows everything but if we want to take only specific fields how would we do

the idea would be

SERV1 
SERV2
SERV3

or user selection

[https://www.mixtecnico.net/github/Captura%20de%20pantalla%202022-03-08%20143112.png](url)

and here as it is visualized in the pdf

[https://www.mixtecnico.net/github/1646760545.pdf](url)

**Comment by rskrzypczak** [comment, marked as answer]
Example:
https://github.com/YetiForceCompany/YetiForceCRM/blob/99592388874ff8eb78600163c1755a48b55d0c22/app/TextParser.php#L39
construction: ModuleName|Columns|Conditions|CustomViewIdOrName|Limit

**Reply by MixTecnico** [reply]
ok but if I want a specific list, only the selected can?

since 'LBL_RECORDS_LIST' only gives me a general listing

example

SERV1 = ID, NAME, TIME
SERV11 = ID, NAME, TIME
SERV23 = ID, NAME, TIME

AND NOT A GENERAL LISTING!

**Reply by MixTecnico** [reply]
@rskrzypczak any ideas?

**Reply by apcloic** [reply]
@MixTecnico
If you want a specific list, you have to specify a custom view ID or name like @rskrzypczak show you.
construction -> ModuleName|Columns|Conditions|**CustomViewIdOrName**|Limit
